swingset/src/main/java/com/nqadmin/swingset/datasources/RowSetOps.java

```diff
@@ -894,4 +894,72 @@ public static void updateColumnObject2(RowSet _rowSet, Object _value, int _colum
 		}
 		
 	}
+
+	/**
+	 * Fetch and return object for specified rowset's specified column;
+	 * type is determined by column's JDBCType.
+	 * @param rowset source rowset
+	 * @param _column source column
+	 * @return 
+	 * @throws SQLException
+	 */
+	public static Object getColumnObject(RowSet rowset, int _column) throws SQLException
+	{
+		if (Boolean.TRUE)
```

Comment:
Won't `if (Boolean.TRUE)` always resolve to true?

Reply:
Right. I think it's OK to replace the 40+ lines by a single line. But that's based on my experience with `H2`; looking at specs and h2 code. I'm not that familiar with other JDBC drivers. I have learned through reading that they can vary greatly. In fact `SQLFeatureNotSupportedException` is listed for almost every method in ResultSet javadoc. So this change leaves both the original and one liner as something that can be easily switched.